eventlivedata/live_db.go

```diff
@@ -0,0 +1,65 @@
+package eventlivedata
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"os"
+	"strconv"
+)
+
+type live_db_adapter interface {
+	initConn() error
+	getLiveHeatMap(event int) (map[string]int, error)
+}
+
+type dynamodb_adapter struct {
```

Comment:
I think in general structs tend to be formatted as `dynamodbAdaptor` (or `DynamodbAdaptor` if public) so not to confuse with package names, which are in snake_case. (Correct me if I'm wrong 😬)